### PR TITLE
I documented the run_docker file. 

### PR DESCRIPTION
**Pull request by PythonLinks**
I have some docker experience, but I did not know what all of those docker commands did, so while waiting for it to build, I went ahead and documented it. 

Often when I approach a new open source project, the first thing I do is to write documentation. 

I hope that helps.

### CODE REVIEW

**Comment by msabramo** [comment]
"Dockermentation" 😆

File: run_docker [hunk 1]
```diff
@@ -1,5 +1,47 @@
 #!/bin/sh
 DOCKER_PORT=8000
-SYSTEM_PORT=8000
+SYSTEM_PORT=8084
```
Comment (mdboom) [inline run_docker]: Now that #355 is merged, let's not change the default here.

**Comment by msabramo** [comment]
I think it's great that you took the initiative to contribute documentation, but I'm not sure about the value of this. I guess the comments here help for someone who is new to Docker who is reading the file, but for people who already know about Docker, the comments might be a bit distracting. Perhaps it's better to just have a comment at the top of the file that links to some Docker documentation page that describes the options that are used?

**Comment by msabramo** [comment]
Oh oops, the comments aren't interspersed. You put them below the command. Hmm, that's not so bad then. Sorry.

**Comment by PythonLinks** [comment]
I will go ahead and add a line below the code, and say that there is documentation only below this line.

**Comment by rth** [comment]
Sorry, closing this. I agree that the docker command by itself may be a bit obscure for users not familiar with docker, but running `docker run --help` should provide most of this information with the added advantage is that it doesn't need maintaining (and we are quite short on maintainers time at present).

**Pull request closed without merging**